bzip2.rs

```diff
@@ -504,40 +503,48 @@ unsafe fn testStream(zStream: *mut FILE) -> bool {
                 unused.as_mut_ptr() as *mut libc::c_void,
                 nUnused,
             );
-            if bzf.is_null() || bzerr != 0 as libc::c_int {
+            if bzf.is_null() || bzerr != 0 {
                 // diverges
                 ioError()
             }
+
+            // there might be multiple files if the input stream is stdin
             streamNo += 1;
-            while bzerr == 0 as libc::c_int {
+
+            while bzerr == 0 {
                 BZ2_bzRead(
                     &mut bzerr,
                     bzf,
                     obuf.as_mut_ptr() as *mut libc::c_void,
-                    5000 as libc::c_int,
+                    5000,
                 );
-                if bzerr == -5 as libc::c_int {
+                if bzerr == libbzip2_rs_sys::BZ_DATA_ERROR_MAGIC {
                     break 'errhandler;
                 }
             }
-            if bzerr != 4 as libc::c_int {
+
+            if bzerr != libbzip2_rs_sys::BZ_STREAM_END {
                 break 'errhandler;
             }
+
+            let mut unusedTmpV: *mut libc::c_void = std::ptr::null_mut::<libc::c_void>();
```

Comment:
```suggestion
            let mut unusedTmpV = std::ptr::null_mut();
```

rustc will infer the right type for calling `BZ2_bzReadGetUnused` and it is casted to a different type later anyway.